.github/workflows/cypress-electron.yml

```diff
@@ -0,0 +1,87 @@
+# Mirrors the Cypress workflow with the following changes:
+# - Cypress run has a spec
+# - We create the stats.json file here since the workflow only runs once.
+name: Cypress Electron
+on:
+  push:
+    branches-ignore:
+      - 'renovate/**'
+  pull_request:
+
+jobs:
+  cypress-run:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        browsers: [electron]
+        node-version: [14.x]
+    services:
+      mongodb:
+        image: mongo:3.6.19
+        ports:
+          - 27017:27017
+      # We need mailhog to catch any emails the api tries to send.
+      mailhog:
+        image: mailhog/mailhog
+        ports:
+          - 1025:1025
```

Comment:
Can we remove this for this workflow?
```suggestion
```

Reply:
Sure.  I'll update the comment about what's not mirrored.